src/sdk.ts

```diff
@@ -95,11 +91,28 @@ export class BadgerSDK {
       network: this.config.network,
       baseURL: this.api.baseURL,
     });
+    this.graph = new BadgerGraph({
+      network: this.config.network,
```

Comment:
should we include the `baseURL: this.graph.baseURL` and expose it? 
i.e. someone is trying to use test subgraphs (also maybe worth considering if we do not use resolve client url on baseURL overrides since they may not conform to our naming convention.

probably not something to worry about rn though.

Reply:
sound reasonable for me, will do